src/Adapter/EntityMapper.php

```diff
@@ -82,7 +82,7 @@ public function load($id, $id_lang, $entity, $entity_defs, $id_shop, $should_cac
                     if ($object_datas_lang = Db::getInstance()->executeS($sql)) {
                         foreach ($object_datas_lang as $row) {
                             foreach ($row as $key => $value) {
-                                if ($key != $entity_defs['primary'] && array_key_exists($key, $entity)) {
+                                if ($key != $entity_defs['primary'] && isset($entity, $key)) {
```

Comment:
Won't work. isset will disable null value, and we sometimes want null values

Reply:
What you want is probably `property_exists`

Reply:
property_exists function bugs with protected properties sadly.